### DISCUSSION TITLE
what does zvl*b means ?

### DISCUSSION THREAD

**Discussion by unicormbu** [iree-org/iree#16279] [Q&A]
there is cmd : 
tools/iree-compile \
  --iree-hal-target-backends=llvm-cpu \
  --iree-llvmcpu-target-triple=riscv64 \
  --iree-llvmcpu-target-abi=lp64d \
  --iree-llvmcpu-target-cpu-features="+m,+a,+f,+d,+zvl512b,+v" \
  --riscv-v-fixed-length-vector-lmul-max=8 \

does v mean rvv1.0 ? 
and zvl 512b means ?

**Comment by dcaballe** [comment]
It's one of the ISA extensions: https://five-embeddev.com/quickref/isa_ext.html
